src/elementPlugins/pluginProvider.tsx

```diff
@@ -57,6 +57,7 @@ class pluginProvider {
 
 function getLocalElementPlugin(folderName: string) {
   const result: PluginComponents = {}
+  // TO-DO: optimize so it only imports the component type (Application, Template, Summary) that is required
```

Comment:
Also another issue to be done? I recommend adding these with the `Epic#6 - Question plugins` tag.

Reply:
The whole pluginProvider needs to be re-written (it's currently just placeholder as it has a hard-coded manifest), so these kind of improvements should be done with that. Have made an issue for the wider re-write: #133

Reply:
Cool. No problem. Just wanted to make sure we don't leave the TODO unattended.